src/main/java/cinema/security/PasswordValidation.java

```diff
@@ -0,0 +1,14 @@
+package cinema.security;
+
+import cinema.security.impl.PasswordValidationImpl;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+
+@Constraint(validatedBy = PasswordValidationImpl.class)
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface PasswordValidation {
```

Comment:
does it work without the required fields?

see your `EmailValidation` constraint